test/JDBC/db_collation_jdbc_schedule

```diff
@@ -0,0 +1,7 @@
+# Schedule File for JDBC Test Framework for local run for database level collation
+# 1. Lines starting with '#' will be treated as comments
+# 2. To run a postgres command: cmd#!#postgresql#!#<enter postgres command>
+# 3. To run a T-SQL command: cmd#!#sqlserver#!#<enter T-SQL command>
+# 4. Keyword "all" is equivalent to running all test files in input folder
+# 5. To add a test, add test name (without extension, ,  and . For example if test file name is TestBigInt.txt write TestBigInt) on a new line
```

Comment:
We are only using this schedule file for mentioning test files to ignore, we should update these comments accordingly.